midtransclient/transactions.py

```diff
@@ -99,15 +99,15 @@ def refundDirect(self, transaction_id,parameters=dict()):
         return response_dict
 
     def notification(self, notification=dict()):
-        is_notification_string = isinstance(notification, str)
+        is_notification_string = isinstance(notification, str if sys.version_info[0] >= 3 else basestring)
```

Comment:
> if sys.version_info[0] >= 3 else basestring
Don't do this, Python 2 is officially deprecated, so we should only focus on Python 3+, no need to support python 2